README.md

```diff
@@ -5,6 +5,8 @@
     <a href="https://travis-ci.org/sonatype-nexus-community/nancy"><img src="https://travis-ci.org/sonatype-nexus-community/nancy.svg?branch=master" alt="Build Status"></img></a>
```

Comment:
We should also cleanup travisci too.

Reply:
Done!

Reply:
I should have been clearer here..... :) we should also delete the .travis.yml as well :) No need to support both CI's.